Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -274,7 +212,18 @@ You can install Cilium on any Kubernetes cluster. Pick one of the options below:
 
        .. code-block:: shell-session
 
-           cilium install --azure-resource-group "${AZURE_RESOURCE_GROUP}"
+           cilium install --azure-resource-group "${AZURE_RESOURCE_GROUP}" \
+             --datapath-mode=tunnel \
```

Comment:
Is `--datapath-mode=tunnel` equivalent to the helm ` --set tunnel=vxlan`?

Reply:
Yea I'm 95% sure `--datapath-mode=vxlan` just sets `DatapathMode` to `DatapathTunnel` by default since `vxlan` is not a valid option for `datapath-mode` 😁

Reply:
@mattstam FYI we have completely revamped `DatapathMode` in https://github.com/cilium/cilium-cli/pull/898 and are adding a new BYOCNI-specific mode in https://github.com/cilium/cilium-cli/pull/899, which will be the basis for the main PR at hand here.